{
  "uid" : "eb56f5683edcd113",
  "name" : "Verify content of text in the Footer",
  "fullName" : "tests.footer_test.TestFooterText#test_fp_02_01_verify_footer_text",
  "historyId" : "a0cc4eafd70d3eb05a1b50599f01724b",
  "time" : {
    "start" : 1728608163725,
    "stop" : 1728608163725,
    "duration" : 0
  },
  "status" : "broken",
  "statusMessage" : "selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\n  (Session info: chrome=129.0.6668.89)\nStacktrace:\n#0 0x55980908502a <unknown>\n#1 0x559808d6b5e0 <unknown>\n#2 0x559808d637eb <unknown>\n#3 0x559808d53a09 <unknown>\n#4 0x559808d55744 <unknown>\n#5 0x559808d53cbd <unknown>\n#6 0x559808d5352f <unknown>\n#7 0x559808d53422 <unknown>\n#8 0x559808d51493 <unknown>\n#9 0x559808d51ada <unknown>\n#10 0x559808d6df41 <unknown>\n#11 0x559808dff8c5 <unknown>\n#12 0x559808ddfb22 <unknown>\n#13 0x559808dfed7d <unknown>\n#14 0x559808ddf8c3 <unknown>\n#15 0x559808dad6b3 <unknown>\n#16 0x559808dae68e <unknown>\n#17 0x55980904fa2b <unknown>\n#18 0x5598090539b1 <unknown>\n#19 0x55980903c225 <unknown>\n#20 0x559809054532 <unknown>\n#21 0x55980902138f <unknown>\n#22 0x559809073f28 <unknown>\n#23 0x5598090740f3 <unknown>\n#24 0x559809083e7c <unknown>\n#25 0x7f77c0294ac3 <unknown>",
  "statusTrace" : "driver = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"58e3f46a158e4c7f09581455523acb03\")>\n\n    @pytest.fixture()\n    @allure.step(f'Open page: {MainPageLinks.URL_MAIN_PAGE}')\n    def main_page_open(driver):\n>       driver.get(MainPageLinks.URL_MAIN_PAGE)\n\ntests/conftest.py:22: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py:353: in get\n    self.execute(Command.GET, {\"url\": url})\n/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py:344: in execute\n    self.error_handler.check_response(response)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.remote.errorhandler.ErrorHandler object at 0x7fb25e40da30>\nresponse = {'status': 500, 'value': '{\"value\":{\"error\":\"unknown error\",\"message\":\"unknown error: net::ERR_CONNECTION_REFUSED\\\\n  ...\\\\n#23 0x5598090740f3 \\\\u003Cunknown>\\\\n#24 0x559809083e7c \\\\u003Cunknown>\\\\n#25 0x7f77c0294ac3 \\\\u003Cunknown>\\\\n\"}}'}\n\n    def check_response(self, response: Dict[str, Any]) -> None:\n        \"\"\"Checks that a JSON response from the WebDriver does not have an\n        error.\n    \n        :Args:\n         - response - The JSON response from the WebDriver server as a dictionary\n           object.\n    \n        :Raises: If the response contains an error message.\n        \"\"\"\n        status = response.get(\"status\", None)\n        if not status or status == ErrorCode.SUCCESS:\n            return\n        value = None\n        message = response.get(\"message\", \"\")\n        screen: str = response.get(\"screen\", \"\")\n        stacktrace = None\n        if isinstance(status, int):\n            value_json = response.get(\"value\", None)\n            if value_json and isinstance(value_json, str):\n                import json\n    \n                try:\n                    value = json.loads(value_json)\n                    if len(value) == 1:\n                        value = value[\"value\"]\n                    status = value.get(\"error\", None)\n                    if not status:\n                        status = value.get(\"status\", ErrorCode.UNKNOWN_ERROR)\n                        message = value.get(\"value\") or value.get(\"message\")\n                        if not isinstance(message, str):\n                            value = message\n                            message = message.get(\"message\")\n                    else:\n                        message = value.get(\"message\", None)\n                except ValueError:\n                    pass\n    \n        exception_class: Type[WebDriverException]\n        e = ErrorCode()\n        error_codes = [item for item in dir(e) if not item.startswith(\"__\")]\n        for error_code in error_codes:\n            error_info = getattr(ErrorCode, error_code)\n            if isinstance(error_info, list) and status in error_info:\n                exception_class = getattr(ExceptionMapping, error_code, WebDriverException)\n                break\n        else:\n            exception_class = WebDriverException\n    \n        if not value:\n            value = response[\"value\"]\n        if isinstance(value, str):\n            raise exception_class(value)\n        if message == \"\" and \"message\" in value:\n            message = value[\"message\"]\n    \n        screen = None  # type: ignore[assignment]\n        if \"screen\" in value:\n            screen = value[\"screen\"]\n    \n        stacktrace = None\n        st_value = value.get(\"stackTrace\") or value.get(\"stacktrace\")\n        if st_value:\n            if isinstance(st_value, str):\n                stacktrace = st_value.split(\"\\n\")\n            else:\n                stacktrace = []\n                try:\n                    for frame in st_value:\n                        line = frame.get(\"lineNumber\", \"\")\n                        file = frame.get(\"fileName\", \"<anonymous>\")\n                        if line:\n                            file = f\"{file}:{line}\"\n                        meth = frame.get(\"methodName\", \"<anonymous>\")\n                        if \"className\" in frame:\n                            meth = f\"{frame['className']}.{meth}\"\n                        msg = \"    at %s (%s)\"\n                        msg = msg % (meth, file)\n                        stacktrace.append(msg)\n                except TypeError:\n                    pass\n        if exception_class == UnexpectedAlertPresentException:\n            alert_text = None\n            if \"data\" in value:\n                alert_text = value[\"data\"].get(\"text\")\n            elif \"alert\" in value:\n                alert_text = value[\"alert\"].get(\"text\")\n            raise exception_class(message, screen, stacktrace, alert_text)  # type: ignore[call-arg]  # mypy is not smart enough here\n>       raise exception_class(message, screen, stacktrace)\nE       selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\nE         (Session info: chrome=129.0.6668.89)\nE       Stacktrace:\nE       #0 0x55980908502a <unknown>\nE       #1 0x559808d6b5e0 <unknown>\nE       #2 0x559808d637eb <unknown>\nE       #3 0x559808d53a09 <unknown>\nE       #4 0x559808d55744 <unknown>\nE       #5 0x559808d53cbd <unknown>\nE       #6 0x559808d5352f <unknown>\nE       #7 0x559808d53422 <unknown>\nE       #8 0x559808d51493 <unknown>\nE       #9 0x559808d51ada <unknown>\nE       #10 0x559808d6df41 <unknown>\nE       #11 0x559808dff8c5 <unknown>\nE       #12 0x559808ddfb22 <unknown>\nE       #13 0x559808dfed7d <unknown>\nE       #14 0x559808ddf8c3 <unknown>\nE       #15 0x559808dad6b3 <unknown>\nE       #16 0x559808dae68e <unknown>\nE       #17 0x55980904fa2b <unknown>\nE       #18 0x5598090539b1 <unknown>\nE       #19 0x55980903c225 <unknown>\nE       #20 0x559809054532 <unknown>\nE       #21 0x55980902138f <unknown>\nE       #22 0x559809073f28 <unknown>\nE       #23 0x5598090740f3 <unknown>\nE       #24 0x559809083e7c <unknown>\nE       #25 0x7f77c0294ac3 <unknown>\n\n/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/errorhandler.py:229: WebDriverException",
  "flaky" : false,
  "newFailed" : false,
  "beforeStages" : [ {
    "name" : "clear_allure_results_folder",
    "time" : {
      "start" : 1728608130626,
      "stop" : 1728608130626,
      "duration" : 0
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  }, {
    "name" : "main_page_open",
    "time" : {
      "start" : 1728608164507,
      "stop" : 1728608164981,
      "duration" : 474
    },
    "status" : "broken",
    "statusMessage" : "selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\n  (Session info: chrome=129.0.6668.89)\nStacktrace:\n#0 0x55980908502a <unknown>\n#1 0x559808d6b5e0 <unknown>\n#2 0x559808d637eb <unknown>\n#3 0x559808d53a09 <unknown>\n#4 0x559808d55744 <unknown>\n#5 0x559808d53cbd <unknown>\n#6 0x559808d5352f <unknown>\n#7 0x559808d53422 <unknown>\n#8 0x559808d51493 <unknown>\n#9 0x559808d51ada <unknown>\n#10 0x559808d6df41 <unknown>\n#11 0x559808dff8c5 <unknown>\n#12 0x559808ddfb22 <unknown>\n#13 0x559808dfed7d <unknown>\n#14 0x559808ddf8c3 <unknown>\n#15 0x559808dad6b3 <unknown>\n#16 0x559808dae68e <unknown>\n#17 0x55980904fa2b <unknown>\n#18 0x5598090539b1 <unknown>\n#19 0x55980903c225 <unknown>\n#20 0x559809054532 <unknown>\n#21 0x55980902138f <unknown>\n#22 0x559809073f28 <unknown>\n#23 0x5598090740f3 <unknown>\n#24 0x559809083e7c <unknown>\n#25 0x7f77c0294ac3 <unknown>\n\n",
    "statusTrace" : "  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/_pytest/fixtures.py\", line 1123, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/_pytest/fixtures.py\", line 902, in call_fixture_func\n    fixture_result = fixturefunc(**kwargs)\n                     ^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/home/runner/work/auto-tests-python/auto-tests-python/tests/conftest.py\", line 22, in main_page_open\n    driver.get(MainPageLinks.URL_MAIN_PAGE)\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py\", line 353, in get\n    self.execute(Command.GET, {\"url\": url})\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py\", line 344, in execute\n    self.error_handler.check_response(response)\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/errorhandler.py\", line 229, in check_response\n    raise exception_class(message, screen, stacktrace)\n",
    "steps" : [ {
      "name" : "Open page: https://brainup.site/",
      "time" : {
        "start" : 1728608164508,
        "stop" : 1728608164980,
        "duration" : 472
      },
      "status" : "broken",
      "statusMessage" : "selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\n  (Session info: chrome=129.0.6668.89)\nStacktrace:\n#0 0x55980908502a <unknown>\n#1 0x559808d6b5e0 <unknown>\n#2 0x559808d637eb <unknown>\n#3 0x559808d53a09 <unknown>\n#4 0x559808d55744 <unknown>\n#5 0x559808d53cbd <unknown>\n#6 0x559808d5352f <unknown>\n#7 0x559808d53422 <unknown>\n#8 0x559808d51493 <unknown>\n#9 0x559808d51ada <unknown>\n#10 0x559808d6df41 <unknown>\n#11 0x559808dff8c5 <unknown>\n#12 0x559808ddfb22 <unknown>\n#13 0x559808dfed7d <unknown>\n#14 0x559808ddf8c3 <unknown>\n#15 0x559808dad6b3 <unknown>\n#16 0x559808dae68e <unknown>\n#17 0x55980904fa2b <unknown>\n#18 0x5598090539b1 <unknown>\n#19 0x55980903c225 <unknown>\n#20 0x559809054532 <unknown>\n#21 0x55980902138f <unknown>\n#22 0x559809073f28 <unknown>\n#23 0x5598090740f3 <unknown>\n#24 0x559809083e7c <unknown>\n#25 0x7f77c0294ac3 <unknown>\n\n",
      "statusTrace" : "  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/home/runner/work/auto-tests-python/auto-tests-python/tests/conftest.py\", line 22, in main_page_open\n    driver.get(MainPageLinks.URL_MAIN_PAGE)\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py\", line 353, in get\n    self.execute(Command.GET, {\"url\": url})\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py\", line 344, in execute\n    self.error_handler.check_response(response)\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/errorhandler.py\", line 229, in check_response\n    raise exception_class(message, screen, stacktrace)\n",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ {
        "name" : "driver",
        "value" : "<selenium.webdriver.chrome.webdriver.WebDriver (session=\"58e3f46a158e4c7f09581455523acb03\")>"
      } ],
      "stepsCount" : 0,
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : true,
      "hasContent" : true
    } ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 1,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : true
  }, {
    "name" : "driver",
    "time" : {
      "start" : 1728608163726,
      "stop" : 1728608164507,
      "duration" : 781
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  } ],
  "afterStages" : [ {
    "name" : "driver::0",
    "time" : {
      "start" : 1728608165013,
      "stop" : 1728608165074,
      "duration" : 61
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  } ],
  "labels" : [ {
    "name" : "epic",
    "value" : "Test Footer"
  }, {
    "name" : "parentSuite",
    "value" : "tests"
  }, {
    "name" : "suite",
    "value" : "footer_test"
  }, {
    "name" : "subSuite",
    "value" : "TestFooter"
  }, {
    "name" : "host",
    "value" : "fv-az523-260"
  }, {
    "name" : "thread",
    "value" : "2358-MainThread"
  }, {
    "name" : "framework",
    "value" : "pytest"
  }, {
    "name" : "language",
    "value" : "cpython3"
  }, {
    "name" : "package",
    "value" : "tests.footer_test"
  }, {
    "name" : "resultFormat",
    "value" : "allure2"
  } ],
  "parameters" : [ ],
  "links" : [ ],
  "hidden" : false,
  "retry" : false,
  "extra" : {
    "severity" : "normal",
    "retries" : [ ],
    "categories" : [ {
      "name" : "Test defects",
      "matchedStatuses" : [ ],
      "flaky" : false
    } ],
    "history" : {
      "statistic" : {
        "failed" : 0,
        "broken" : 4,
        "skipped" : 0,
        "passed" : 248,
        "unknown" : 0,
        "total" : 252
      },
      "items" : [ {
        "uid" : "ee400efc907fbba4",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/322//#testresult/ee400efc907fbba4",
        "status" : "passed",
        "time" : {
          "start" : 1728556849596,
          "stop" : 1728556849641,
          "duration" : 45
        }
      }, {
        "uid" : "fc8e2c33fc17b7a",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/321//#testresult/fc8e2c33fc17b7a",
        "status" : "passed",
        "time" : {
          "start" : 1728553780545,
          "stop" : 1728553780597,
          "duration" : 52
        }
      }, {
        "uid" : "36a6dd67dff8f48f",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/148//#testresult/36a6dd67dff8f48f",
        "status" : "passed",
        "time" : {
          "start" : 1728521876989,
          "stop" : 1728521877038,
          "duration" : 49
        }
      }, {
        "uid" : "2317fc34fd67e0d2",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/320//#testresult/2317fc34fd67e0d2",
        "status" : "passed",
        "time" : {
          "start" : 1728470348727,
          "stop" : 1728470348778,
          "duration" : 51
        }
      }, {
        "uid" : "a3edba0faf6af5f7",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/319//#testresult/a3edba0faf6af5f7",
        "status" : "passed",
        "time" : {
          "start" : 1728467757184,
          "stop" : 1728467757244,
          "duration" : 60
        }
      }, {
        "uid" : "2ddc5d01fce6c177",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/147//#testresult/2ddc5d01fce6c177",
        "status" : "passed",
        "time" : {
          "start" : 1728435475004,
          "stop" : 1728435475055,
          "duration" : 51
        }
      }, {
        "uid" : "2222f2ca567e528e",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/318//#testresult/2222f2ca567e528e",
        "status" : "passed",
        "time" : {
          "start" : 1728377872278,
          "stop" : 1728377872334,
          "duration" : 56
        }
      }, {
        "uid" : "82b5f0e45ff42738",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/317//#testresult/82b5f0e45ff42738",
        "status" : "passed",
        "time" : {
          "start" : 1728375074509,
          "stop" : 1728375074553,
          "duration" : 44
        }
      }, {
        "uid" : "6b53d7f7c48b94c3",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/316//#testresult/6b53d7f7c48b94c3",
        "status" : "passed",
        "time" : {
          "start" : 1728374961677,
          "stop" : 1728374961732,
          "duration" : 55
        }
      }, {
        "uid" : "b6e583c188307121",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/146//#testresult/b6e583c188307121",
        "status" : "passed",
        "time" : {
          "start" : 1728349054040,
          "stop" : 1728349054097,
          "duration" : 57
        }
      }, {
        "uid" : "37801da58f1dbee3",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/315//#testresult/37801da58f1dbee3",
        "status" : "passed",
        "time" : {
          "start" : 1728292633360,
          "stop" : 1728292633408,
          "duration" : 48
        }
      }, {
        "uid" : "a0cd3568ecda7093",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/314//#testresult/a0cd3568ecda7093",
        "status" : "passed",
        "time" : {
          "start" : 1728289675681,
          "stop" : 1728289675728,
          "duration" : 47
        }
      }, {
        "uid" : "f506a791c9425c71",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/145//#testresult/f506a791c9425c71",
        "status" : "passed",
        "time" : {
          "start" : 1728263082026,
          "stop" : 1728263082071,
          "duration" : 45
        }
      }, {
        "uid" : "988436f4ecba7113",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/313//#testresult/988436f4ecba7113",
        "status" : "passed",
        "time" : {
          "start" : 1728226187905,
          "stop" : 1728226187955,
          "duration" : 50
        }
      }, {
        "uid" : "901b1a0940f99f32",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/312//#testresult/901b1a0940f99f32",
        "status" : "passed",
        "time" : {
          "start" : 1728221333139,
          "stop" : 1728221333193,
          "duration" : 54
        }
      }, {
        "uid" : "6ee5b467a410a1a9",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/144//#testresult/6ee5b467a410a1a9",
        "status" : "passed",
        "time" : {
          "start" : 1728176591088,
          "stop" : 1728176591136,
          "duration" : 48
        }
      }, {
        "uid" : "cb35f7287923d8ff",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/311//#testresult/cb35f7287923d8ff",
        "status" : "passed",
        "time" : {
          "start" : 1728117261496,
          "stop" : 1728117261544,
          "duration" : 48
        }
      }, {
        "uid" : "8e4e12c93c14a48d",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/310//#testresult/8e4e12c93c14a48d",
        "status" : "passed",
        "time" : {
          "start" : 1728108663798,
          "stop" : 1728108663854,
          "duration" : 56
        }
      }, {
        "uid" : "2b50bafcd16b5380",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/143//#testresult/2b50bafcd16b5380",
        "status" : "passed",
        "time" : {
          "start" : 1728089923698,
          "stop" : 1728089923743,
          "duration" : 45
        }
      }, {
        "uid" : "63d3f0cef7a30d0d",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/142//#testresult/63d3f0cef7a30d0d",
        "status" : "passed",
        "time" : {
          "start" : 1728003519702,
          "stop" : 1728003519747,
          "duration" : 45
        }
      } ]
    },
    "tags" : [ ]
  },
  "source" : "eb56f5683edcd113.json",
  "parameterValues" : [ ]
}